madanalysis/layout/plotflow.py

```diff
@@ -1198,193 +1181,549 @@ def DrawMATPLOTLIB(
+                if stackmode:
+                    backstyle3 = [3004, 3005, 3006]
+                    backstyle = backstyle3[ind]
+                    backcolor = linecolor3[ind]
+            elif len(histos) == 4:
+                linecolor4 = [9, 46, 8, 4]
+                linecolor = linecolor4[ind]
+                if stackmode:
+                    backstyle4 = [3004, 3005, 3006, 3007]
+                    backstyle = backstyle4[ind]
+                    backcolor = linecolor4[ind]
+            elif len(histos) == 5:
+                linecolor5 = [9, 46, 8, 4, 6]
+                linecolor = linecolor5[ind]
+                if stackmode:
+                    backstyle5 = [3004, 3005, 3006, 3007, 3013]
+                    backstyle = backstyle5[ind]
+                    backcolor = linecolor5[ind]
+            elif len(histos) == 6:
+                linecolor6 = [9, 46, 8, 4, 6, 2]
+                linecolor = linecolor6[ind]
+                if stackmode:
+                    backstyle6 = [3004, 3005, 3006, 3007, 3013, 3017]
+                    backstyle = backstyle6[ind]
+                    backcolor = linecolor6[ind]
+            elif len(histos) == 7:
+                linecolor7 = [9, 46, 8, 4, 6, 2, 7]
+                linecolor = linecolor7[ind]
+                if stackmode:
+                    backstyle7 = [3004, 3005, 3006, 3007, 3013, 3017, 3022]
+                    backstyle = backstyle7[ind]
+                    backcolor = linecolor7[ind]
+            elif len(histos) == 8:
+                linecolor8 = [9, 46, 8, 4, 6, 2, 7, 3]
+                linecolor = linecolor8[ind]
+                if stackmode:
+                    backstyle8 = [3004, 3005, 3006, 3007, 3013, 3017, 3022, 3315]
+                    backstyle = backstyle8[ind]
+                    backcolor = linecolor8[ind]
+            elif len(histos) == 9:
+                linecolor9 = [9, 46, 8, 4, 6, 2, 7, 3, 42]
+                linecolor = linecolor9[ind]
+                if stackmode:
+                    backstyle9 = [3004, 3005, 3006, 3007, 3013, 3017, 3022, 3315, 3351]
+                    backstyle = backstyle9[ind]
+                    backcolor = linecolor9[ind]
+            elif len(histos) == 10:
+                linecolor10 = [9, 46, 8, 4, 6, 2, 7, 3, 42, 48]
+                linecolor = linecolor10[ind]
+                if stackmode:
+                    backstyle10 = [
+                        3004,
+                        3005,
+                        3006,
+                        3007,
+                        3013,
+                        3017,
+                        3022,
+                        3315,
+                        3351,
+                        3481,
                     ]
-                ),
-                axis=1,
+                    backstyle = backstyle10[ind]
+                    backcolor = linecolor10[ind]
+            else:
+                linecolor = self.color
+                self.color += 1
```

Comment:
Can we think about improving a bit what is above|?

Reply:
I haven't touched the colour flow yet; they are the same as before. What kind of improvement are you thinking?

Reply:
Maybe something slightly more compact.... Because those things are very heavy in terms of number of lines. This is of course a very minor thing to modify.